docs/start/latest-updates.mdx

```diff
@@ -0,0 +1,139 @@
+- You can now use the experimental option in [`qiskit_ibm_runtime.options.EstimatorOptions`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EstimatorOptions) to enable the Probabilistic Error Amplification (PEA) error mitigation method.
+
+#### Upgrade notes
+
+- Qiskit SDK v1.1 is now supported and required.
+
+#### Bug fixes
+
+- Fixed an issue where retrieved jobs would only return `RuntimeJob` instances, even if the job was run with a V2 primitive. Now, V2 primitive jobs will be returned correctly as `RuntimeJobV2` instances.
+- Fixed the combination of `insert_multiple_cycles` and `coupling_map` options in [`PadDynamicalDecoupling`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.transpiler.passes.scheduling.PadDynamicalDecoupling). This combination allows to select staggered dynamical decoupling with multiple sequence cycles in each delay that crosses the threshold set by `sequence_min_length_ratios`.
+- Fixed measurement twirling docstring that incorrectly indicated it’s enabled by default for Sampler.
+- Fixed issues related to target generation and calibration data.
+
+See the full [0.24.0 release notes](/api/qiskit-ibm-runtime/release-notes#0240-2024-06-10).
+{/* use url format /api/qiskit-ibm-runtime/release-notes#0XYZ to link to the specific 0.XY.Z version of the release notes */}
+
+### Qiskit Transpiler Service Client v0.3.0
+
+We’re pleased to share that the beta release of the Qiskit Transpiler Service is now available to all IBM Quantum&trade; Premium Plan users.
+
+The Qiskit Transpiler Service leverages the resources of IBM Cloud&reg; to provide users with the latest transpilation capabilities from the Qiskit SDK. It offers a Python library that helps users seamlessly integrate the service into their current Qiskit patterns and workflows. Perhaps most importantly, the service invites users to experiment with new and improved AI-powered transpiler passes — cutting-edge tools that might be faster and produce better results than traditional transpilation methods.
+
+On the IBM Quantum blog, we take a deep dive into the new beta release with a special focus on the new AI-powered passes. The blog includes detailed explanations showing how to use the Qiskit Transpiler Service for both full circuit transpilation and standalone AI-powered passes, with code examples for each. [Take a look.](https://www.ibm.com/quantum/blog/ai-transpiler-passes)
+
+{/* no other release notes available yet */}
+
+## Qiskit IBM Runtime service updates
+
+This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use. 
+
+### 28 March 2024
+
+Qiskit Runtime primitives now support QASM 3 as the input format for circuits and standard JSON output format. See [Qiskit IBM Runtime REST API](/api/runtime) for examples.
```

Comment:
I think the official name is `OpenQasm 3`.

Reply:
Actually OpenQASM 3 - good catch